source/_posts/2019-10-25-rip-hassbian.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "R.I.P Hassbian"
+description: "The end of Hassbian"
+date: 2019-10-25 01:11:03
+date_formatted: "October 25, 2019"
+author: Fredrik Lindqvist
+author_twitter: Landrash
+comments: true
+categories: community
+---
+
+It's time for the part of Home Assistant that once was the simplest way to install Home Assistant to retire. Hassbian is being retired.
```

Comment:
> It's time for the part of Home Assistant that once was the simplest way to install Home Assistant to retire. Hassbian is being retired.

Feels clunky. Maybe:

_It's time for what was once the simplest way to install Home Assistant to retire._

Reply:
Love it. Thank you :)